packages/gatsby-plugin-image/src/components/hooks.ts

```diff
@@ -145,17 +143,26 @@ export function getPlaceholderProps(
   const wrapperStyle: CSSProperties = {}
 
   if (backgroundColor) {
+    wrapperStyle.backgroundColor = backgroundColor
+
     if (layout === `fixed`) {
       wrapperStyle.width = width
       wrapperStyle.height = height
+      wrapperStyle.backgroundColor = backgroundColor
+      wrapperStyle.position = `relative`
+    } else if (layout === `constrained`) {
```

Comment:
ℹ️ I've decided to explicitly show the different placeholder styles between layout types. I find it helpful to quickly visualize what is the style of a dedicated placeholder directly